client/store/RootState.ts

```diff
@@ -1,5 +1,7 @@
+import { User } from '@core/meta/user'
 import { AssessmentState } from './assessment/stateType'
 
 export type RootState = {
   assessment: AssessmentState
+  user: User
```

Comment:
user might be undefined

Reply:
hmm, also assessment before initialization... should we define it as partial then?